scripts/requirements.txt

```diff
@@ -0,0 +1,2 @@
+botocore==1.8.36
```

Comment:
What's the purpose of this file?

Reply:
This is to specify the Python dependencies

Reply:
With the build script automation.sh do we need this file anymore?